extension/pgtelemetry--1.5.sql--1.6.sql

```diff
@@ -0,0 +1,19 @@
+-- WAL telemetry
+
+create function wal_telemetry_create_or_select_record()
```

Comment:
Actually this is upgrade script and therefore we need to replace `wal_telemetry_create_or_select_record()` otherwise we won't be able to upgrade `pgtelemetry`.
Can you replace it into `create or replace function`?